### PR TITLE
add default app name if metadata.yaml does not exist

### PR DESCRIPTION
**Pull request by jdkandersson**

### CODE REVIEW

File: .github/workflows/integration_test.yaml [hunk 1]
```diff
@@ -131,7 +131,7 @@ jobs:
         run: bash ${{ inputs.pre-run-script }}
       - name: Run integration tests
         run: |
-          echo "CHARM_NAME=$(yq '.name' metadata.yaml)" >> $GITHUB_ENV
+          echo "CHARM_NAME=$(yq '.name' metadata.yaml | echo UNKNOWN)" >> $GITHUB_ENV
```
Comment (arturo-seijas) [inline .github/workflows/integration_test.yaml:134]: This will output also an error code. Can we check for the existence of the file instead?

**Review by arturo-seijas — CHANGES_REQUESTED** [review]

File: .github/workflows/integration_test.yaml (hunk 1 above)
Comment (arturo-seijas) [inline .github/workflows/integration_test.yaml:134]: This will output also an error code. Can we check for the existence of the file instead?

Comment (jdkandersson) [reply .github/workflows/integration_test.yaml]: Thank you, fixed

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by merkata — APPROVED** [review]
Nice, approving.

**Pull request merged**